bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/DefaultSystemChannelTypeProvider.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.thing.internal;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+
+import org.eclipse.smarthome.core.thing.type.ChannelType;
+import org.eclipse.smarthome.core.thing.type.ChannelTypeUID;
+import org.eclipse.smarthome.core.thing.type.SystemChannelTypeProvider;
+
+/**
+ * Implementation providing default system wide channel types and mechanisms for
+ * registration/unregistration.
+ * 
+ * @author Ivan Iliev - Initial Contribution
+ * 
+ */
+public class DefaultSystemChannelTypeProvider implements SystemChannelTypeProvider {
+
+    /**
+     * Signal strength default system wide {@link ChannelType}
+     */
+    private static final ChannelType SYSTEM_CHANNEL_SIGNAL_STRENGTH = new ChannelType(new ChannelTypeUID(
+            "system:signal-strength"), false, "Number", "Signal Strength", null, "QualityOfService",
```

Comment:
The idea was to simply filter out such Items by a tag, which are more QoS related rather can covering functional aspects. But I agree that we first should discuss it again. So @kolkoo could you please remove it for this PR.